packages/utils/src/lib/transform.ts

```diff
@@ -156,3 +157,56 @@ export function toOrdinal(value: number): string {
   return `${value}th`;
 }
 /* eslint-enable no-magic-numbers */
+
+export class AuditsNotImplementedError extends Error {
+  constructor(list: { slug: string }[], auditSlugs: string[]) {
+    super(
+      `audits: "${auditSlugs
+        .filter(slug => !list.some(a => a.slug === slug))
+        .join(', ')}" not implemented`,
+    );
+  }
+}
+
+export function filterByAuditSlug<
```

Comment:
This is hard for me to navigate. It includes a lot of one-letter types. Does the notation need to be so complicated? I am no TypeScript expert but I wouldn't know how to debug this in the future on my own.

Reply:
I think know what Michael was trying to do, if the audit's `slug` field as a union type then this function could have nice autocomplete. But unfortunately the slugs are strings anyway, so I believe the generics are unnecessary, as mentioned here: https://github.com/code-pushup/cli/pull/472#discussion_r1480144213